_pages/en_US/installing-boot9strap-(smilehax-iie-jpn).txt

```diff
@@ -0,0 +1,98 @@
+{% endcapture %}
+<details>{{ technical_info | markdownify }}</details>
+{: .notice--info}
+
+### What You Need 
+* The game "SmileBASIC" installed on your console
+    * If you have downloaded it before, you can [redownload it](https://en-americas-support.nintendo.com/app/answers/detail/a_id/607/~/how-to-download-or-redownload-content-in-nintendo-3ds-eshop)
+    * Your SD card must be inserted in your console to install SmileBASIC
+* The latest release of [smilehax-IIe](https://github.com/zoogie/smilehax-IIe/releases/download/v1.1/Release_sh2e_v1.1.zip) (direct download) 
+* The latest release of [SafeB9SInstaller](https://github.com/d0k3/SafeB9SInstaller/releases/download/v0.0.7/SafeB9SInstaller-20170605-122940.zip) (direct download)
+* The latest release of [boot9strap](https://github.com/SciresM/boot9strap/releases/download/1.4/boot9strap-1.4.zip) (direct download)
+* The latest release of [nimdsphax](https://github.com/luigoalma/nimdsphax/releases/download/v1.0/nimdsphax_v1.0.zip) (direct download)
+* The latest release of [Luma3DS](https://github.com/LumaTeam/Luma3DS/releases/latest) (the Luma3DS `.zip` file)
+* The latest release of [HBL otherapps](https://github.com/zoogie/ninjhax2.x/releases/download/v11.17/otherapps_with_CfgS_11.17.zip) (direct download)
+
+### Instructions
+
+#### Section I - Prep Work
+1. Make sure your copy of SmileBASIC is updated to the latest version
+1. Power off your console
+1. Insert your SD card into your computer
+1. Copy `boot.firm` and `boot.3dsx` from the Luma3DS `.zip` to the root of your SD card
+    + The root of the SD card refers to the initial directory on your SD card where you can see the Nintendo 3DS folder, but are not inside of it
+1. Create a folder named `boot9strap` on the root of your SD card
+1. Copy `boot9strap.firm` and `boot9strap.firm.sha` from the boot9strap `.zip` to the `/boot9strap/` folder on your SD card
+1. Copy `SafeB9SInstaller.bin` from the SafeB9SInstaller `.zip` to the root of your SD card
+1. Copy the otherapp payload for your model/region/version from `otherapps_with_CfgS_11.17.zip` to the root of your SD card, then rename the payload to `otherapp.bin`
+    + Do not add the `.bin` extension if you do not already see it
+1. Transfer the `.wav` for your region from the Release_sh2e `.zip` to a device that can play `.wav` files, for example a phone
+1. Copy the petitcom `.icn` for your region to the root of your SD card
+1. Create a folder named `3ds` on the root of your SD card if it does not already exist
+    + This folder stores homebrew applications and data; it is different from the `Nintendo 3DS` folder that the console automatically generates
+1. Copy the `nimdsphax` folder from the nimdsphax `.zip` to the `/3ds/` folder on your SD card
+1. To downgrade your Smilebasic to an exploitable ver (3.3.2), rename the file at:
```

Comment:
Please test your changes. its all one line.
I highly suggest you follow the general style from the guide.
So use navigate, ->, and code formatting. In addition, you should keep capitalization consistent and, do we actually need to tell the guide follower that they will downgrade to an exploitable version? if so, definititely s/ver/version, and remove (3.3.2) as noone cares